scripts/zones/Dynamis-Valkurm/mobs/Vanguard_Prelate.lua

```diff
@@ -1,21 +1,14 @@
 -----------------------------------
 -- Area: Dynamis Valkurm
+--  MOB: Vanguard_Prelate
 -----------------------------------
------------------------------------
-require("scripts/globals/status")
-mixins = { require("scripts/mixins/dynamis_beastmen") }
-
------------------------------------
--- onMobSpawn Action
+mixins = {require("scripts/mixins/dynamis_beastmen")}
+mixins = {require("scripts/mixins/job_special")}
```

Comment:
this not going to work like that, it'll replace the dynamis mixin with the 2hour one (which is why I said the merge was gonna be annoying heh).  So far I only got the dynamis listener on Dynamis Valkurm mobs, but it's probably ready to be added to all of them at this point.. but you'd have to also set the currency var if someone were to do that

Reply:
```lua
mixins = {require("scripts/mixins/dynamis_beastmen"), require("scripts/mixins/job_special")}
```